app/assets/javascripts/discourse/app/models/bookmark.js

```diff
@@ -125,9 +125,20 @@ const Bookmark = RestModel.extend({
     ).capitalize();
   },
 
-  @discourseComputed("linked_post_number", "fancy_title", "topic_id")
-  topicLink(linked_post_number, fancy_title, id) {
-    return Topic.create({ id, fancy_title, linked_post_number });
+  @discourseComputed()
```

Comment:
Do we need this to be a computed property when it is computing against nothing?

Reply:
Yes, I tried without it being a computed property and it was not accessible from the HBS view.